job/common/src/main/java/alluxio/job/wire/TaskInfo.java

```diff
@@ -46,26 +49,40 @@
   private Serializable mResult;
   private long mLastUpdated;
   private String mWorkerHost;
+  private String mDescription;
 
   /**
    * Default constructor.
    */
-  public TaskInfo() {}
+  public TaskInfo() {
+    mErrorMessage = "";
+    mDescription = "";
+  }
 
   /**
    * Constructs a new TaskInfo from jobId, taskId, Status, and workerAddress.
    * @param jobId the job id
    * @param taskId the task id
    * @param status the status
    * @param workerAddress the worker address
+   * @param args the arguments
```

Comment:
can you add what the args can be?

Reply:
It's currently any Serializable object that any Task takes.